### DISCUSSION TITLE
32비트 버전은 어디에 있습니까?

### DISCUSSION THREAD

**Discussion by peacharrow** [KadeArchive/Kade-Engine#395] [Q&A] [answered]
Where's the 32-bit version? I use the PC version of Windows 10 32bit, so please reply if you know it.

**Comment by puyoxyz** [comment, marked as answer]
CI doesn't build for 32 bit but you can build it yourself by adding `-32` to the build command (i.e. `lime build windows -32`), see [BUILDING.md](https://github.com/KadeDev/Kade-Engine/blob/master/BUILDING.md) for info on how to build.
